frontends/concrete-python/concrete/fhe/compilation/module.py

```diff
@@ -169,6 +177,10 @@ def encrypt(
             Optional[Union[Value, Tuple[Optional[Value], ...]]]:
                 encrypted argument(s) for evaluation
         """
+
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
no need to explain? like "# In simulate_encrypt_run_decrypt:, we skip encryption and decryption, and run will be normal python execution